templates/torrc.j2

```diff
@@ -80,207 +50,65 @@ OutboundBindAddressExit {{ torrc_outbound_bind_address_exit }}
-## the default exit policy. Leave commented to just use the default, which is
-## described in the man page or at
-## https://www.torproject.org/documentation.html
-##
-## Look at https://www.torproject.org/faq-abuse.html#TypicalAbuses
-## for issues you might encounter if you use the default exit policy.
-##
-## If certain IPs and ports are blocked externally, e.g. by your firewall,
-## you should update your exit policy to reflect this -- otherwise Tor
-## users will be told that those destinations are down.
-##
-## For security, by default Tor rejects connections to private (local)
-## networks, including to the configured primary public IPv4 and IPv6 addresses,
-## and any public IPv4 and IPv6 addresses on any interface on the relay.
-## See the man page entry for ExitPolicyRejectPrivate if you want to allow
-## "exit enclaving".
-##
-#ExitPolicy accept *:6660-6667,reject *:* # allow irc ports on IPv4 and IPv6 but no more
-#ExitPolicy accept *:119 # accept nntp ports on IPv4 and IPv6 as well as default exit policy
-#ExitPolicy accept *4:119 # accept nntp ports on IPv4 only as well as default exit policy
-#ExitPolicy accept6 *6:119 # accept nntp ports on IPv6 only as well as default exit policy
-#ExitPolicy reject *:* # no exits allowed
-#}
+
 {%- for policy in torrc_exit_policies %}
 ExitPolicy {{ policy }}
 {% endfor %}
-{#
-## Bridge relays (or "bridges") are Tor relays that aren't listed in the
-## main directory. Since there is no complete public list of them, even an
-## ISP that filters connections to all the known Tor relays probably
-## won't be able to block all the bridges. Also, websites won't treat you
-## differently because they won't know you're running Tor. If you can
-## be a real relay, please do; but if not, be a bridge!
-##
-## Warning: when running your Tor as a bridge, make sure than MyFamily is
-## NOT configured.
-#BridgeRelay 1
-## By default, Tor will advertise your bridge to users through various
-## mechanisms like https://bridges.torproject.org/. If you want to run
-## a private bridge, for example because you'll give out your bridge
-## address manually to your friends, uncomment this line:
-#PublishServerDescriptor 0
-#}
+
 {%- if torrc_bridge_relay %}
+{# Rise warning if MyFamiliy is set? #}
 BridgeRelay {{ renderbool(torrc_bridge_relay) }}
 PublishServerDescriptor {{ renderbool(torrc_publish_server_descriptor) }}
 {% endif %}
-{#
-## Configuration options can be imported from files or folders using the %include
-## option with the value being a path. If the path is a file, the options from the
-## file will be parsed as if they were written where the %include option is. If
-## the path is a folder, all files on that folder will be parsed following lexical
-## order. Files starting with a dot are ignored. Files on subfolders are ignored.
-## The %include option can be used recursively.
-#%include /etc/torrc.d/
-#%include /etc/torrc.custom
-#}
+
 {%- for include in torrc_includes %}
```

Comment:
This can provoke that some variable (for example, `torrc_ipv6_exit`) can be defined multiple times. What's the behavior of tor in this case? It refuses to load the configuration file or it gets the latest one?